src/types.ts

```diff
@@ -12,12 +12,25 @@ export type RepoConfig = {
   uuid?: string;
 };
 
+type CustomRule = {
+  rule: (
+    pullRequestInfo: {
+      pullRequest: BB.PullRequest;
+      buildStatuses: BB.BuildStatus[];
+      approvals: string[];
```

Comment:
jfmi - What do we store - the name? the number of approvals?

Reply:
List of the usernames of the approvers